### PR TITLE
Update testing-webhooks.md - Round #1

### PR DESCRIPTION
**Pull request by Berealady9**
More of the same.

I don't think I've seen the "SEEALSO" passage like the one on line 15 before, so hopefully that's formatted correctly.

### CODE REVIEW

File: sfcasts/testing-webhooks.md [hunk 1]
```diff
@@ -1,146 +1,80 @@
...
-[Zenstruck Foundry](https://github.com/zenstruck/foundry) we already have
-installed to reset and manage the test database automatically.
+Our test is failing because we need to set up a test database. We 
+*could* do this manually using Doctrine console commands, but let's take advantage of [Zenstruck Foundry](https://github.com/zenstruck/foundry), which we've already installed, to reset and manage the test database *automatically*.
 
-In your test class, add `use ResetDatabase`. This also
-takes care of cleaning up the database between tests, so no worries about
-duplicate email errors. Try running the test again, and this time it
-passes. Great! Now, let's write an actual test.
+In the test class, add `use ResetDatabase`. This also cleans up the database between tests, so we don't have to worry about duplicate email errors. If we run the test again... it passed this time! Great! Now, let's write an *actual* test.
```
Comment (bocharsky-bw) [inline sfcasts/testing-webhooks.md:44]:
```suggestion
In the test class, add `use ResetDatabase`. This also cleans up the database between tests, so we don't have to worry about duplicate email errors. Sweet! If we run the test again... it passed this time! Great! Now, let's write an *actual* test.
```

File: sfcasts/testing-webhooks.md [hunk 2]
```diff
@@ -1,146 +1,80 @@
...
 
-Next, let's simulate an actual `POST` request to the webhook endpoint:
+We can copy the JSON payload from the Ngrok web interface (if you still have that running), *or* we can copy it from the LemonSqueezy dashboard under "Webhooks". Copy the whole request block and, back in our code, in `tests/`, create a new directory. Call it `fixtures`, and inside *that*, create a new file. Call *this one* `order_created.json`, and... *paste*.
 
-`$client->request('POST', '/webhook/lemon-squeezy', [], [], [], $json);`
+In our test, above the request, say `$json = file_get_contents(__DIR__.'/../fixtures/order_created.json')`. At the end, add another assert - `$this->assertNotNull()` - and... whoops! I forgot to create a `$user` variable above, so let's fix that. Now, pass `$user->getCustomerId()` as the argument to our new assert call, and for the error message, say `LemonSqueezy customer ID not set!`. Finally, add `$this->assertEquals(1000001, $user->getCustomerId(), 'LemonSqueezy customer ID mismatch!')`. Whew! Testing time!
```
Comment (bocharsky-bw) [inline sfcasts/testing-webhooks.md:54]:
```suggestion
In our test, above the request, say `$json = file_get_contents(__DIR__.'/../fixtures/order_created.json')`. At the end, add another assert - `$this->assertNotNull()` - and... whoops! I forgot to create a `$user` variable above, so let's fix that. Now, pass `$user->getLsCustomerId()` as the argument to our new assert call, and for the error message, say `LemonSqueezy customer ID not set!`. Finally, add `$this->assertEquals(1000001, $user->getLsCustomerId(), 'LemonSqueezy customer ID mismatch!')`. Whew! Testing time!
```

File: sfcasts/testing-webhooks.md [hunk 3]
```diff
@@ -1,146 +1,80 @@
...
-are trying to send a fake request there.
+Whoops! Well, that's expected. We copied and pasted the payload string, but it looks like it might be a little different from the original, so its signature isn't valid anymore.
 
-So, we could skip the signature check completely in this spot by injecting
-the Symfony environment and add `if ($this-env === 'test')` then just return.
+If we take a look at the request parser... *yep*. We added this `verifySignature()` method to *prevent* unauthorized requests to the endpoint, but now *we're* the ones trying to send a fake request there. How the turns have tabled!
 
-But I don't like this workaround, better solution would be to sign the
-requests in your test.
+We could skip the signature check here *completely* by injecting the Symfony environment, adding `if ($this-env === 'test')`, and simply returning. But I don't like this workaround. A *better* solution would be to *sign* the requests in your test so they appear to be legitimate.
```
Comment (bocharsky-bw) [inline sfcasts/testing-webhooks.md:70]:
```suggestion
We could skip the signature check here *completely* by injecting the Symfony environment, adding `if ($this->env === 'test')`, and simply returning. But I don't like this workaround. A *better* solution would be to *sign* the requests in your test so they appear to be legitimate.
```

**Review by bocharsky-bw — APPROVED** [review]
Great! And yes, that `SEEALSO` is our internal syntax so it should work well, but thanks for the heads-up!
